optimize.sh

```diff
@@ -4,20 +4,45 @@ command -v shellcheck > /dev/null && shellcheck "$0"
 
 export PATH=$PATH:/root/.cargo/bin
 
-# This parameter allows us to mount a folder into docker container's "/code"
-# and build "/code/contracts/mycontract".
-contractdir="$1"
-echo "Building contract in $(realpath -m "$contractdir")"
+mkdir -p artifacts
+contractdir=${1:-.}
```

Comment:
With the Dockerfile in this PR, $1 is always set, right? Then we can avoid defining the default value in two places

Reply:
Sure, I guess you are right

Reply:
We can also leave the default value here and allow calling optimize.sh  with no arg. We just should decide where the default is defined.